File: airbyte-cdk/java/airbyte-cdk/datastore-mongo/src/main/java/io/airbyte/cdk/db/mongodb/MongoDatabaseException.java
/*
 * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 */

package io.airbyte.cdk.db.mongodb;

public class MongoDatabaseException extends RuntimeException {

  public static final String MONGO_DATA_BASE_NOT_FOUND = "Data Base with given name - %s not found.";

  public MongoDatabaseException(final String databaseName) {
    super(String.format(MONGO_DATA_BASE_NOT_FOUND, databaseName));
  }

}

File: airbyte-cdk/java/airbyte-cdk/datastore-mongo/bin/main/io/airbyte/cdk/db/mongodb/MongoDatabaseException.kt
/*
 * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 */
package io.airbyte.cdk.db.mongodb

class MongoDatabaseException(databaseName: String?) :
    RuntimeException(String.format(MONGO_DATA_BASE_NOT_FOUND, databaseName)) {
    companion object {
        const val MONGO_DATA_BASE_NOT_FOUND: String = "Data Base with given name - %s not found."
    }
}
